addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14436,7 +14436,7 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Activate Teletext" with label #23050
 #: system/settings/settings.xml
 msgctxt "#36174"
-msgid "Enable Teletext when watching a live TV stream."
+msgid "Enable teletext when watching a live TV stream."
```

Comment:
Well I dont mind that, but http://www.oxforddictionaries.com/definition/english/teletext?q=Teletext

Ill change it anyway.